pkg/spanconfig/spanconfig.go

```diff
@@ -40,13 +40,9 @@ type ReconciliationDependencies interface {
 	// applicable to a given tenant.
 	Accessor
 
-	// TODO(zcfgs-pod): We'll also needs access to a tenant's system.descriptor,
-	// or ideally a Watcher emitting relevant Updates from a tenant's
-	// system.{descriptor,zones}. That suggests we'll want two Watcher
-	// implementations:
-	// - Something per-store, watching over system.span_configurations.
-	// - Something watching over a tenant's system.{descriptor,zones}, emitting
-	//   updates for the reconciliation process.
+	// SQLWatcher maintains a rangefeed over system.{descriptor, zones} and emits
```

Comment:
Don't include "rangefeed", which is an impl detail. Let's use the comments from our API sketch offline.